test/__specs__/hello-world.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { test, expect } from '@playwright/test'
+
+test('Hello World test', async ({ page }) => {
+  await page.goto('/hello-world-app')
+  let heading = await page.locator('h1')
+  const appFontWeight = await heading.evaluate((element) => {
+    return window.getComputedStyle(element).fontWeight
+  })
+  await page.goto('/hello-world-page')
+  heading = await page.locator('h1')
+  const pageFontWeight = await heading.evaluate((element) => {
+    return window.getComputedStyle(element).fontWeight
```

Comment:
Nice thing about playwright is you are getting the entire page -  just like a browser. So `getComputedStyles` is real world and contains all the properties the element has on the page. 

In my jest test, this wasn't the case. 

I think `getComputedStyles` or something similar is how I would write the server style registry test.

Reply:
Yeah, jest uses `jsdom` as the virtual DOM, which has some [unimplemented APIs](https://github.com/jsdom/jsdom?tab=readme-ov-file#unimplemented-parts-of-the-web-platform). I think using playwright to use more of the DOM features is the right way to go here (and not just about these types of features, but for actually testing browser interactions).

Reply:
> In my jest test, this wasn't the case.

I think there's a misunderstanding here about what Jest does out of the box. Maybe we should discuss what our testing goals are so we can decide what tool we want to use?